sql/moz-fx-data-shared-prod/firefox_ios_derived/attributable_clients_v1/query.sql

```diff
@@ -0,0 +1,57 @@
+CREATE TEMP FUNCTION sum_map_values(map ARRAY<STRUCT<key STRING, value INT64>>)
```

Comment:
We _should_ move this to a persistent UDF, since we're using it in multiple places now. Maybe `mozfun.map.sum_values`?

Reply:
That sounds fair, but perhaps this should be a separate PR?

Reply:
Created DENG-882 as a follow-up for doing this.